integration_tests/package.json

```diff
@@ -2,24 +2,26 @@
   "description": "Quay Integration Testing",
```

Comment:
We should really actually move all this into the root `package.json`, but that can be another PR.

Reply:
Sure, it is OK for me, I will create another PR later